hw/top_englishbreakfast/data/top_englishbreakfast.hjson

```diff
@@ -158,24 +158,47 @@
-      { name: "por_io_div2", gen: true,  type: "top", parent: "por_aon", clk: "io_div2" }
-      { name: "por_io_div4", gen: true , type: "top", parent: "por_aon", clk: "io_div4" }
-      { name: "por_usb",     gen: true,  type: "top", parent: "por_aon", clk: "usb"     }
-      { name: "lc",          gen: true,  type: "top", parent: "lc_src",  clk: "main"    }
-      { name: "lc_io_div4",  gen: true,  type: "top", parent: "lc_src",  clk: "io_div4" }
-      { name: "sys",         gen: true,  type: "top", parent: "sys_src", clk: "main"    }
-      { name: "sys_io_div4", gen: true,  type: "top", parent: "sys_src", clk: "io_div4" }
-      { name: "sys_aon",     gen: true,  type: "top", parent: "sys_src", clk: "aon"     }
-      { name: "spi_device",  gen: true,  type: "top", parent: "sys_src", clk: "io_div2", sw: true }
-      { name: "spi_host0",   gen: true,  type: "top", parent: "sys_src", clk: "io",      sw: true }
-      { name: "usb",         gen: true,  type: "top", parent: "sys_src", clk: "usb",     sw: true }
+      { name: "por_aon",     gen: false, type: "top",                    clock: "aon"     }
+      { name: "lc_src",      gen: false, type: "int",                    clock: "io_div4" }
+      { name: "sys_src",     gen: false, type: "int",                    clock: "io_div4" }
+      { name: "por",         gen: true,  type: "top", parent: "por_aon", clock: "main"    }
+      { name: "por_io",      gen: true,  type: "top", parent: "por_aon", clock: "io"      }
+      { name: "por_io_div2", gen: true,  type: "top", parent: "por_aon", clock: "io_div2" }
+      { name: "por_io_div4", gen: true , type: "top", parent: "por_aon", clock: "io_div4" }
+      { name: "por_usb",     gen: true,  type: "top", parent: "por_aon", clock: "usb"     }
+      { name: "lc",          gen: true,  type: "top", parent: "lc_src",  clock: "main"    }
+      { name: "lc_io_div4",  gen: true,  type: "top", parent: "lc_src",  clock: "io_div4" }
+      { name: "sys",         gen: true,  type: "top", parent: "sys_src", clock: "main"    }
+      { name: "sys_io_div4", gen: true,  type: "top", parent: "sys_src", clock: "io_div4" }
+      { name: "sys_aon",     gen: true,  type: "top", parent: "sys_src", clock: "aon"     }
+      { name: "spi_device",  gen: true,  type: "top", parent: "sys_src", clock: "io_div2", sw: true }
+      { name: "spi_host0",   gen: true,  type: "top", parent: "sys_src", clock: "io",      sw: true }
+      { name: "usb",         gen: true,  type: "top", parent: "sys_src", clock: "usb",     sw: true }
     ]
   }
+  // The reset requests: peripherals are deduced, but internal and debug need
+  // to be configure in hjson file.
+  reset_requests: {
+    int: [
+      {
+        "name": "MainPwr",
+        "desc": "main power glitch reset request",
+        "module": "pwrmgr_aon"
+      },
+      {
+        "name": "Esc",
+        "desc": "escalation reset request",
+        "module": "alert_handler"
```

Comment:
Huh, but englishbreakfast doesn't have an alert_handler. Does this field not actually do anything?

Reply:
TBD: notice the earlgrey and englishbreakfast use the same toplevel.sv.tpl, so there are plenty of questions about this.